entries/jQuery.effects.saveStyle.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<entries>
+	<entry type="method" name="jQuery.effects.saveStyle" added="1.12">
+		<title>jQuery.effects.saveStyle()</title>
+		<signature>
+			<argument name="element" type="jQuery">
+				<desc>The element to save styles for.</desc>
+			</argument>
+		</signature>
+		<desc>Saves all inline styles applied to an element.</desc>
+		<longdesc>
+			<p>Saves all inline styles applied to an element. This is useful when animating various styles and restoring the existing styles at the end. The saved styles can be restored using <a href="/jQuery.effects.restoreStyle/"><code>jQuery.effects.restoreStyle()</code></a>.</p>
```

Comment:
Its not just internal details. The semantics of the method were unclear to me until I read the code, which makes for bad documentation.